pkg/test/resourcefixture/testdata/basic/bigquery/v1beta1/bigquerytable/bigquerytable-externaldata/update.yaml

```
@@ -0,0 +1,52 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: bigquery.cnrm.cloud.google.com/v1beta1
+kind: BigQueryTable
+metadata:
+  name: bigquerytable-${uniqueId}
+spec:
+  resourceID: bigquerytable-${uniqueId}
+  datasetRef:
+    name: bigquerydataset-${uniqueId}
+  friendlyName: BigQuery Table Updated
+  description: An updated BigQuery table.
+  schema: >
+    [
+      {"name": "dateField", "type": "DATE", "mode": "REQUIRED"},
+      {"name": "intField", "type": "INTEGER", "mode": "REQUIRED"},
+      {"name": "stringField1", "type": "STRING", "mode": "NULLABLE"},
+      {"name": "stringField2", "type": "STRING", "mode": "NULLABLE"},
+      {"name": "stringField3", "type": "STRING", "mode": "NULLABLE"}
+    ]
+  expirationTime: 2064442632000
+  externalDataConfiguration:
+    autodetect: false
+    sourceUris:
+      - gs://cloud-samples-data/bigquery/us-states/us-states-by-date.csv
```

would it be possible to have another file with different delimiter and other options to fully test update.

---

Yes, but probably not now.

1. Under gs://cloud-samples-data/bigquery/, I didn't find a csv file with a different delimiter. Considering this is likely a sample folder owned by bigquery, I don't think we should add any data to it. We probably need to create a sample data bucket by ourselves.
2. I actually updated all the fields in the `csvOptions` in update.yaml, even if the delimiter and possibly other options don't match what's in the source file, but the request was still successfully. I'm guessing we either didn't use the right configuration, or we used it but the error was not returned by PUT.

So I prefer we spend the time for a better test case after we understand the situation.